### ISSUE TITLE
TreeView API for converters should be introduced

### ISSUE THREAD

**Issue by pjasiun** [ckeditor/ckeditor5-core#164]
Is should be simple to wrap some text into an inline style which will be automatically merged with siblings:

```
<b>foo</b>bar<i><b>bom</b></i>
->apply <b> on bar
<b>foobar<i>bom</i></b>
```

It may be done by the additional class: `treeView.Writter`.

**Comment by pjasiun** [comment]
During the meeting we decided that TreeView elements will have a property which will say:

 - Is it block (<p>), inline (<b>) or inline-block  (<img>) element,
 - what is the priority (for inline),
 - can two elements can be merged (if they have different priority they can not).

**Comment by pjasiun** [comment]
Also we decided that switching order of elements will be a future task, not handled by this ticket, so for the example in the fist post it will be:

`<b>foobar</b><i><b>bom</b></i>`

**Comment by pjasiun** [comment]
We decided that block will be called `container`, inline will be called `attribute` and we do not need inline-block.

**Comment by pjasiun** [comment]
Closed by #245.
